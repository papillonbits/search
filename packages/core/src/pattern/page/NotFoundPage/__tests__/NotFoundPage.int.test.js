import { async, regular } from './NotFoundPage.int.story'

jest.mock('react-router-dom', () => ({ Link: () => <div>some link</div> }))

describe('<NotFoundPage />', () => {
  afterEach(() => jest.clearAllMocks())

  describe('Render', () => {
    test('must match async', () => {
      expect(global.renderToJSON(async())).toMatchSnapshot()
    })

    test('must match regular', () => {
      expect(global.renderToJSON(regular())).toMatchSnapshot()
    })
  })
})
