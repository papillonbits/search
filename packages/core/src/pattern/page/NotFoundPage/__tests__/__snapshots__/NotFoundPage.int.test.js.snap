// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFoundPage /> Render must match async 1`] = `<div />`;

exports[`<NotFoundPage /> Render must match regular 1`] = `
<div>
  <div
    class="alert"
  >
    <div
      class="flash-messages"
    >
      <div
        class="flash flash-error"
      >
        <mock-file
          aria-hidden="true"
          classname="mr-2"
          height="16"
          role="img"
          style="display: inline-block; fill: currentColor; user-select: none; vertical-align: text-bottom;"
          viewbox="0 0 16 16"
          width="16"
        />
        No match for route. Return to 
        <div>
          some link
        </div>
      </div>
    </div>
  </div>
</div>
`;
